realm-transformer/src/main/kotlin/io/realm/transformer/RealmTransformer.kt

```diff
@@ -154,8 +153,17 @@ class RealmTransformer(val project: Project) : Transform() {
             val targetSdk: String? = project.getTargetSdk()
             val minSdk: String?  = project.getMinSdk()
             val sync: Boolean = Utils.isSyncEnabled(project)
-            val analytics = RealmAnalytics(packages, containsKotlin, sync, targetSdk, minSdk)
-            analytics.execute()
+            val app = project.plugins.findPlugin("com.android.application") != null
```

Comment:
Nitpick. I would check for each case specifically and if we couldn't detect either app or library use "unknown"?